super_dragon_magical_unicorns_by_disney_fish/views/edit.ejs

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Info</title>
+  </head>
+  <body>
+    <h1>Edit info about <%= dragon[0].name %> the dragon here</h1>
```

Comment:
Again, using findOne or sending dragon[0] to the view will prevent having it from being an array on the view. Not a big problem, but a nice little clean change you could make.